tests/travis/install-db2.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -ex
+
+echo Setting up IBM DB2
+
+sudo docker pull ibmcom/db2express-c:10.5.0.5-3.10.0
+sudo docker run \
+    -d \
+    -p 50000:50000 \
+    -e DB2INST1_PASSWORD=Doctrine2018 \
+    -e LICENSE=accept \
+    --name db2 \
+    ibmcom/db2express-c:10.5.0.5-3.10.0 \
+    db2start
+
+sleep 60
```

Comment:
This may be too much but `sleep 5` became not enough when I added creation of the tablespace. We could try looking for a more elegant solution.